Authentication_and_Authorization/hello-security/login-controller.php

```diff
@@ -6,19 +6,45 @@
 <body>
 <a href="index.php"> Take me home, country roads ... </a> <br>
 <?php
+
 include "db-connection.php";
+
+function getHashedPassword($conn, $email){
+
+    $sqlQueryForPassword = "SELECT password FROM person where email = '" . $email . "';";
+
```

Comment:
Here you don't need to use double quotation marks. You can remove them if you want.

Reply:
Double quotation marks are needed because without them the request would look like this:  SELECT * from person where email = teacher@email;, which is an invalid request. A correct query should look like this:
SELECT * from person where email = 'teacher@email';, which I achieve through double quotation marks.

Reply:
I mean that code works if you use only single quotation marks. For example:   $sqlQueryForPassword = "SELECT password FROM person where email = ' . $email . ';";

Reply:
Sorry for the misunderstanding, I thought you suggested to remove both types of quotes. But your example is also incorrect because in this case we will treat dots as dots, not as concatenation operator. Your example will work if we remove dots. For example: $sqlQueryForPassword = "SELECT password FROM person where email = '$email';";